File: PHP/Baju.php
<?php
include "Aksesoris.php";
class Baju extends Aksesoris{
    private $untuk = "";
    private $size = "";
    private $merk = "";
    function __construct($id, $nama, $harga, $stok, $foto, $jenis, $bahan, $warna, $untuk, $size, $merk){
        $this->setID($id);
        $this->setNamaProduk($nama);
        $this->setHargaProduk($harga);
        $this->setStokProduk($stok);
        $this->setFotoProduk($foto);
        $this->setJenis($jenis);
        $this->setBahan($bahan);
        $this->setWarna($warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }
    function setUntuk($untuk){
        $this->untuk = $untuk;
    }
    function setSize($size){
        $this->size = $size;
    }
    function setMerk($merk){
        $this->merk = $merk;
    }
    function getUntuk(){
        return $this->untuk;
    }
    function getSize(){
        return $this->size;
    }
    function getMerk(){
        return $this->merk;
    }
}
?>

File: PYTHON/Baju.py
#untuk, size, merk

from Aksesoris import Aksesoris

class Baju(Aksesoris):
    __untuk = ""
    __size = ""
    __merk = ""

    def __init__(self):
        self.__id = ""
        self.__nama_produk = ""
        self.__harga_produk = 0
        self.__stok_produk = 0
        self.__jenis = ""
        self.__bahan = ""
        self.__warna = ""
        self.__untuk = ""
        self.__size = ""
        self.__merk = ""

    def setUntuk(self, untuk):
        self.__untuk = untuk
    def setSize(self, size):
        self.__size = size
    def setMerk(self, merk):
        self.__merk = merk

    def getUntuk(self):
        return self.__untuk
    def getSize(self):
        return self.__size
    def getMerk(self):
        return self.__merk